### ISSUE TITLE
Documentation extension

### ISSUE THREAD

**Issue by tdipisa** [georchestra/mapstore2-georchestra#67]
An extension must be provided to reach the online documentation related to the viewer and its installed extensions.

- The URL of the documentation must be parametrizable by the administrator in backoffice.  

![image](https://user-images.githubusercontent.com/1280027/76071818-24c7dd80-5f97-11ea-9446-7f97a74fb13b.png)

**Comment by offtherailz** [comment]
This can be implemented by extending the [HelpLink](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/plugins/HelpLink.jsx) plugin.

Actually the URL can be overridden only by using ConfigUtils.getConfigProp("docsUrl"). This may be a global config and therefore is not so flexible. 

It should also be possible to override it in a better way (by configuring the HelpLink plugin) by merging this functionality:
https://github.com/geosolutions-it/MapStore2/pull/4911/files#diff-1a598a860b645277bc475e3248cf7466R214

Because of the low priority I suggest to postpone this issue after #4911 merge
